neural_compressor/torch/algorithms/weight_only/gptq.py

```diff
@@ -690,13 +698,13 @@ def tmp(_, inp, out):
                 # Step 2.6: export to compressed model
                 for layer_name in sequential_layers:
                     weight_config_this_layer = self.get_layer_config(self.get_full_layer_name(layer_name, block_idx))
-                    gptq_scale = gptq_config[self.get_full_layer_name(layer_name, block_idx)]["scale"]
+                    gptq_scale = gptq_config[self.get_full_layer_name(layer_name, block_idx)]["scale"].cpu()

```

Comment:
Why do we give up supporting quantization on GPUs?

Reply:
This is not correct logic.  weight is already moved to cpu three line above. Logic is like below:
![image](https://github.com/user-attachments/assets/f2822a98-991b-45b9-8b17-a57997931d01)

Reply:
If we move the pack logic to device, HPU will allocate unexpected memory (not resolved). So I move it to cpu for all device since the CPU already provides objective speed.